lib/services/project-templates-service.ts

```diff
@@ -46,5 +46,21 @@ export class ProjectTemplatesService implements IProjectTemplatesService {
 		this.$logger.trace(`Using NativeScript verified template: ${templateName} with version ${version}.`);
 		return this.$npmInstallationManager.install(templateName, projectDir, { version: version, dependencyType: "save" });
 	}
+
+	private getTemplateNameToBeTracked(templateName: string, realTemplatePath: string): string {
```

Comment:
Not sure, but maybe this should be in `try { } catch { }`. In case something fails here we don't want to stop the command execution.

Reply:
Nice idea, I've added the try-catch and test for this case.